src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -567,5 +568,9 @@
 			"\tif the clicked button is enchantment option 1:",
 			"\t\tset the applied enchantments to sharpness 10 and unbreaking 10")
 		.since("2.5");
+		Skript.registerEvent("Inventory Pickup", SimpleEvent.class, InventoryPickupItemEvent.class, "inventory pick[ ]up")
+			.description("Called when an inventory (a hopper, a hopper minecart, etc.) picks up an item")
```

Comment:
When making line breaks, Java standards require two indentations not a single indentation. You can look at the other events.